### ISSUE TITLE
--resource-tags are not propagated to all resources created by copilot

### ISSUE THREAD

**Issue by foragerr** [aws/copilot-cli#2766]
Besides ECS resources, copilot makes S3 buckets and lambda functions. Tags supplied via `--resource-tags` flag are not applied to these scaffolding resources. 

In my case, our organization requires certain tags on all resources for cost accounting. Untagged resources are marked for deletion. 

1. Feature Request: Propagate `--resource-tags` to all taggable resources including scaffolding
2. Workaround: Is there a straightforward workaround like editing the generated cloudformation templates? Will they get overwritten on the next copilot run

**Comment by huanjani** [comment]
Related: #696

**Comment by huanjani** [comment]
Hi @foragerr! 

May I ask which command you're using with the `--resource-tags` flag?

**Comment by huanjani** [comment]
I just tried to replicate the issue.

I ran `copilot app init --resource-tags ...` and the tags _did_ trickle down to Lambda and S3.
When I created another service with `copilot svc init` and ran `copilot svc deploy --resource-tags ...`, the new tag was applied to the new Lambda function for that service (along with the tag created with `copilot app init`). It was not added, however, to the Lambda function and S3 bucket created  previously. 

The current behavior makes sense to me, but I can see how people may be caught by surprise-- we'll try to make the docs more specific! 

Thank you for bringing this to our attention!
